source/features/linkify-user-location.tsx

```diff
@@ -3,47 +3,35 @@ import select from 'select-dom';
 import {wrap} from '../libs/dom-utils';
 import features from '../libs/features';
 
-function addLocation(baseElement?: HTMLElement): void {
-	const locationIcon = select('.octicon-location', baseElement)!;
-
-	if (!locationIcon) {
+function addLocation(baseElement: HTMLElement): void {
+	const location = select('.octicon-location', baseElement)?.nextSibling;
+	if (!location) {
 		return;
 	}
 
-	const location = locationIcon.parentElement!.textContent!.trim();
-	const googleMapsLink = `https://www.google.com/maps/place/${encodeURIComponent(location)}/`;
+	const locationName = location.textContent!.trim();
+	const googleMapsLink = `https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(locationName)}`;
 
-	wrap(locationIcon.nextElementSibling || locationIcon.nextSibling!, <a href={googleMapsLink} />);
+	wrap(location, <a href={googleMapsLink} className="text-gray" />);
 }
 
 const hovercardObserver = new MutationObserver(([mutation]) => {
 	addLocation(mutation.target as HTMLElement);
 });
 
 function init(): void | false {
-	if (features.isUserProfile()) {
-		for (const profileElement of select.all('.js-profile-editable-area')) {
-			addLocation(profileElement);
-		}
-	}
-
-	if (features.isOrganizationProfile()) {
-		addLocation(select('.orghead')!);
-	}
+	addLocation(document.body);
```

Comment:
On user profile page there are two containers for all profile information, and their visibility is toggled with some media query. We need to linkify locations in both elements.

Reply:
Let’s put a for-of loop in addLocation, it lets also skip the first condition in that function